server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -229,4 +229,41 @@ public void onFailure(Exception e) {
             }
         });
     }
+
+    public void testDeleteOperations() throws Exception {
+        final String nodeA = internalCluster().startNode();
+        final String nodeB = internalCluster().startNode();
+
+        createIndex(
+            INDEX_NAME,
+            Settings.builder()
```

Comment:
Sorry I missed this on the last PR.  You shouldn't need to set these settings here because we override the indexSettings() method at the top of this file.

```
        createIndex(INDEX_NAME);

```
Will create the index with the default settings we override.

Reply:
Yeah sorry I too missed it